charts/devlake/templates/ingresses.yaml

```diff
@@ -63,8 +63,12 @@ spec:
           - path: /{{ include "devlake.grafanaEndpointPrefix" . }}
           {{- end }}
             {{- if semverCompare ">=1.18-0" $.Capabilities.KubeVersion.GitVersion }}
+            {{- if .Values.ingress.useDefaultNginx }}
+            pathType: ImplementationSpecific
```

Comment:
This should be good enough, but why not making the pathType itself a variable?

Reply:
in fact, I just copy paste exactly what was working before:
```
              {{- if semverCompare ">=1.19-0" $.Capabilities.KubeVersion.GitVersion }}
              service:
                name: {{ .Values.grafana.ingressServiceName | default ( include "grafana.fullname" (dict "Values" .Values.grafana "Chart" (dict "Name" "grafana") "Release" .Release ) ) }}
                port:
                  number: {{ .Values.grafana.ingressServicePort | default .Values.grafana.service.port }}
              {{- else }}
              serviceName: {{ .Values.grafana.ingressServiceName | default ( include "grafana.fullname" (dict "Values" .Values.grafana "Chart" (dict "Name" "grafana") "Release" .Release ) ) }}
              servicePort: {{ .Values.grafana.ingressServicePort | default .Values.grafana.service.port }}
              {{- end }}
```